README.md

```diff
@@ -1,2 +1,351 @@
-# year-cohort-projectName (eg: 22-jul-projectName)
-project template repo
+# Recipes API
+
+## API documentation
+
+### Return/Create user userId - story-2
+
+- **URL**
+
+  /users
+
+- **Method:**
+
+  `POST`
+
+- **URL Params**
+
+  none
+
+  **Example:**
+
+  `/users`
+
+## get user
+
+- **Success Response:**
+  - **Code:** 200
+  - **Content:**
+
+```json
+{
+  "message": "Successfully retrieved user",
+  "data": {
+    "userId": "1"
+  }
+}
+```
+
+- **Error Response:**
+
+- **Code:** 500 SERVER ERROR
+- **Content:** `{"message": "Unexpected error", "data": [] }`
+
+#### add user if not existing
+
+- **Success Response:**
+  - **Code:** 201
+  - **Content:**
+
+```json
+{
+  "message": "Successfully created user",
+  "data": {
+    "userId": "1"
+  }
+}
+```
+
+- **Error Response:**
+
+  - **Code:** 500 SERVER ERROR
+    **Content:** `{"message": "Unexpected error"}`
+
+## Return all recipes for a user - story3
+
+- **URL**
+
+  /users/:userId/recipes
+
+- **Method:**
+
+  `GET`
+
+- **URL Params**
+
+  none
+
+  **Example:**
+
+  `/users/1/recipes`
+
+- **Success Response:**
+- **Code:** 200
+- **Content:**
+
+```json
+{
+  "message": "Successfully retrieved all recipes",
+  "data": [
+    {
+      "id": 1,
+      "name": "baps",
+      "instructions": "how to make baps",
```

Comment:
Are you going to return the full instructions here? Wouldn't that get really really long?

Reply:
instructions removed!